net-ftp/lftp/lftp-4.7.7.recipe

```diff
@@ -62,19 +63,15 @@ GLOBAL_WRITABLE_FILES="
 	settings/lftp.conf keep-old
 	"
 
-PATCH()
-{
-	sed -i configure.ac -e 's|^AM_CONFIG_HEADER|AC_CONFIG_HEADERS|g'
-}
-
 BUILD()
 {
 	libtoolize --force --copy --install
 	aclocal -I m4
 	autoconf
 	automake
-	runConfigure --omit-dirs binDir ./configure --with-openssl --enable-static \
-		--bindir=$prefix/bin
+	runConfigure --omit-dirs binDir ./configure --with-openssl \
+		 --enable-static --bindir=$prefix/bin \
+		CPPFLAGS=-I/boot/system/develop/headers/readline
```

Comment:
I mean: on x86_gcc2h this path is invalid.
BTW usually using readline headers happen with: 
#include <readline/readline.h>
So if you try to add this path to the include paths, there might be something wrong with the include macro, and you might want to patch this one instead. Hope it helps.